scripts/bump-version.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+if [[ "$1" != "" ]]; then
+  VERSION=$1
+elif [[ "$BRANCH" =~ ^release\/v.*$ ]]; then
+  VERSION=${BRANCH#*release/v}
+else
+  echo "$BRANCH does not appear to be a release branch, please specify VERSION manually"
+  echo "$(basename $0) <version-number>"
+  exit 1
+fi
+
+echo Bumping the version number to $VERSION
+sed -i '' "s/package_version = \".*\"/package_version = \"$VERSION\"/" main.go
+sed -i '' "s/\"version\": \".*\"/\"version\": \"$VERSION\"/" package.json
+sed -i '' "s/## TBD/## $VERSION ($(date '+%Y-%m-%d'))/" CHANGELOG.md
```

Comment:
The changelog style is currently h1 (`#`) for the version/date line - in fact, most changelogs have a single h1 `# Changelog` and then h2's for each version. Might be worth matching that style rather than changing this.

Reply:
> Might be worth matching that style rather than changing this.

I'd rather the changelog matched our normal style - each version being an h2, with a single h1 "Changelog"